skins/metfi-dao.scss

```
@@ -0,0 +1,10 @@
+.mantra-dao {
```

The css selector should similar to the skin file name. `skin.scss` `.skin {}`.
```suggestion
.metfi-dao {
```